django_airavata/apps/workspace/static/django_airavata_workspace/js/containers/ProjectListContainer.vue

```diff
@@ -5,6 +5,7 @@
                 <h1 class="h4 mb-4">Browse Projects</h1>
             </div>
             <div id="col-new-project" class="col">
+                <add-file-button></add-file-button>
```

Comment:
Can you put this below the `project-button-new` component and add a class of `mt-1`?